src/configs/configureMobX.ts

```diff
@@ -0,0 +1,8 @@
+import { configure } from 'mobx';
+
+configure({
+  useProxies: 'ifavailable',
+  computedRequiresReaction: true,
+  reactionRequiresObservable: true,
+  observableRequiresReaction: true,
+});
```

Comment:
По идее в configs мы храним конфиги, но ничего не конфигурируем. То есть файл можно переименовать в mobxConfig, в нем оставить сам конфиг. А вызывать `configure(config)` уже наверху в main.ts, где ты подключаешь глобальные стили
```suggestion
const config = {
  useProxies: 'ifavailable',
  computedRequiresReaction: true,
  reactionRequiresObservable: true,
  observableRequiresReaction: true,
};
```

Reply:
done